src/app/models/publiccomment.ts

```diff
@@ -0,0 +1,32 @@
+import { Comment } from './comment';
+import { Document } from './document';
+import { Project } from './project';
+
+export class PublicComment {
+  _id: number;
+  isPublished: boolean;
+  dateCompleted: Date;
+  dateStarted: Date;
+  openHouses: any;
+  additionalText: string;
+  informationLabel: string;
+  relatedDocuments: [Document];
```

Comment:
It looks like we're using multiple different styles to declare arrays:
```
arrayA: Array<SomeType>
arrayB: [SomeType]
arrayC: SomeType[]
```
I know @btzc had some issues with some of the above declarations and picked the one that seemed to work, but I'd like to understand what is different between one and another as they should be equivalent, and maybe pick one for consistency.
Calling in @RandyInVictoria  and @asanchezr who have more experience with AngularIO and TypeScript

Reply:
Fixed. Changed all to Array<SomeType>